app/javascript/controllers/map_controller.js

```diff
@@ -128,6 +114,11 @@ export default class extends Controller {
     document.addEventListener("mouseup", () => {
       target.dataset.beingDragged = false
```

Comment:
This is being deleted right below this. Is it still necessary?

Reply:
Thanks! Removed in 55c9060